.github/workflows/npm-publish.yml

```diff
@@ -20,3 +21,42 @@ jobs:
       - run: npm publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.npm_token }}
+
+  publish-docker:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      # https://github.com/docker/login-action/tree/v1.8.0#docker-hub
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # https://github.com/manics/action-major-minor-tag-calculator-test
+      - name: Get list of tags
+        id: gettags
+        # TODO: Move to org?
+        uses: manics/action-major-minor-tag-calculator@main
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          prefix: "manics/action-major-minor-tag-calculator-test:"
+
+      - name: Display tags
+        run: echo "Docker tags ${{ steps.gettags.outputs.tags }}"
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: images/${{ matrix.image }}
+          platforms: linux/amd64,linux/arm64
+          push: ${{ startsWith(github.ref, 'refs/tags/') }}
+          tags: ${{ join(fromJson(steps.gettags.outputs.tags)) }}
```

Comment:
Could you add a comment explaining what is done here / what kind of string can be expected to come from this expression?